From: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>
Date: Sun, 15 Oct 2023 14:04:31 +0200
Subject: tests: Remove unused variable in pgp.c

Signed-off-by: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>
---
 tests/pgp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/pgp.c b/tests/pgp.c
index bc167eb..5d0bdb6 100644
--- a/tests/pgp.c
+++ b/tests/pgp.c
@@ -296,7 +296,6 @@ test_pgp_message_and_files (void)
   g_autofree char *attachment_1_savepath = NULL;
   g_autofree char *attachment_2_savepath = NULL;
   g_autofree char *attachment_3_savepath = NULL;
-  g_autofree char *mime_sha1 = NULL;
   g_autofree char *signed_and_encrypted_msg = NULL;
   g_autofree char *content_message = NULL;
   g_autofree char *save_directory = NULL;
